circle.yml

```diff
@@ -19,12 +19,10 @@ machine:
   java:
     version: openjdk8
   post:
-    # Manually install very recent version of cassandra; unless we create a user we have to run it as root
-    # uncomment below when datastax has version 3.9 available. at time of commit, latest was 3.7
-    #- curl -SL http://www-us.apache.org/dist/cassandra/3.9/apache-cassandra-3.9-bin.tar.gz | tar xz
-    - curl -SL http://cassci.datastax.com/job/trunk/lastSuccessfulBuild/artifact/build/apache-cassandra-3.10-SNAPSHOT-bin.tar.gz | tar xz
-    - sed -i -e 's/INFO/WARN/g' apache-cassandra-*/conf/logback.xml
-    - sudo ./apache-cassandra-*/bin/cassandra -R
+    # Manually install very recent version of cassandra
+    - curl -SL http://downloads.datastax.com/community/dsc-cassandra-3.0.8-bin.tar.gz | tar xz
```

Comment:
Sure thing, will check tonight

On Tue, Sep 6, 2016, 4:11 PM Adrian Cole <notifications@github.com> wrote:

> In circle.yml
> <https://github.com/openzipkin/zipkin/pull/1252#discussion_r77641755>:
>
> > @@ -19,12 +19,10 @@ machine:
> >    java:
> >      version: openjdk8
> >    post:
> > -    # Manually install very recent version of cassandra; unless we create a user we have to run it as root
> > -    # uncomment below when datastax has version 3.9 available. at time of commit, latest was 3.7
> > -    #- curl -SL http://www-us.apache.org/dist/cassandra/3.9/apache-cassandra-3.9-bin.tar.gz | tar xz
> > -    - curl -SL http://cassci.datastax.com/job/trunk/lastSuccessfulBuild/artifact/build/apache-cassandra-3.10-SNAPSHOT-bin.tar.gz | tar xz
> > -    - sed -i -e 's/INFO/WARN/g' apache-cassandra-*/conf/logback.xml
> > -    - sudo ./apache-cassandra-*/bin/cassandra -R
> > +    # Manually install very recent version of cassandra
> > +    - curl -SL http://downloads.datastax.com/community/dsc-cassandra-3.0.8-bin.tar.gz | tar xz
>
> @abesto <https://github.com/abesto> not sure if this died because of the
> out-of-memory warning we are getting in circleci, or if I somehow made a
> syntax error I can't see.. can you give second eyes?
> https://circleci.com/gh/openzipkin/zipkin/138
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openzipkin/zipkin/pull/1252/files/a8325ab4e4881fb973f4809a28c222db91c08c3f..dbcd44559e8cd0db2864a309761efa07bbd231a3#r77641755>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADqToBe8Zjy4FabQTRCeWGTyPuMwlvZks5qnXRbgaJpZM4Jrhum>
> .
>
-- 
Zoltán Nagy
https://abesto.net